Trimertrip file creation description

bergazin opened the pull request:
I fixed the bullet points in one of CB's README's, added a host folder to the trimertrip directory and included a file generation description in the readme for the trimertrip host (here https://github.com/MobleyLab/SAMPL7/commit/db3fcfd02eea1eb3cbcfd872d436a3c892cc852b)

bergazin commented:
Added all of the guest input files (PDB, SDF, MOL2 format) and the jupyter notebook that was used to generate them to the directory. Mentioned that the host is achiral in the readme.

davidlmobley commented on host_guest/GDCC_and_guests/README.md:
```diff
@@ -35,5 +35,4 @@ For positively charged guests, chloride salts are expected to be used.
 # What's here
 
 All files here so far are extremely tentative pending finalization.
```
I think they are actually final now, yes?

bergazin replied:
I believe so (unless there are any email threads I've missed)

davidlmobley commented on host_guest/GDCC_and_guests/README.md:
```diff
@@ -35,5 +35,4 @@ For positively charged guests, chloride salts are expected to be used.
 All files here so far are extremely tentative pending finalization.
-- Hosts: `OA.pdb` and `exoOA.pdb` provided by Paolo Suating (Tulane) April 26, 2019; `SAMPL7_tentative_hosts.cdx` provided by Bruce Gibb April 26, 2019.
-- Guests: `SAMPL7_tentative_guests.cdx` provided by Bruce Gibb April 26, 2019.
+- `hosts_guest_files/`: Directory containing Chemdraw (`.cdx`), PDB, MOL2 and SDF files.
```
Why did you put these under another subdirectory? I suppose could be OK, but then leaves the higher-level directory nearly empty.

Also, why `hosts_guest_files` as if there are many hosts and one guest? Probably I'd call it `host_guest_files`.

bergazin replied:
I added the cdx files up a directory, but I think the host PDBs should stay in the subdirectory because that's where the guest PDBs are stored. Will rename the folder

davidlmobley replied:
I guess I was asking, why not have all the files in the base directory for `GDCC_and_guests`?

Either way, you need to end up with consistent organization across all three host-guest systems.

davidlmobley commented on host_guest/Isaacs_clip/README.md:
```diff
@@ -15,3 +15,5 @@ Following this, the challenge is expected to launch around June 1, 2019, with ex
 
 All files here so far are extremely tentative pending finalization.
```
I think we can remove the tentative from this AND from teh file names.

davidlmobley commented on host_guest/Isaacs_clip/host_guest/README.md:
```diff
@@ -0,0 +1,9 @@
+
+- Provided is the Trimertrip host for this challenge in PDB, Tripos MOL2, and SDF file format (trimertrip.mol2, trimertrip.sdf, trimertrip.pdb).
+- Trimertrip host files were created as follows:
+  - Original starting structure (in `CIF` file format, which will be added here once the challenge ends) was converted to a PDB using OpenEye toolkits. Water, a guest, a single Na+ ion, and connectivity records were manually removed. OpenEye toolkit was used to normalize the molecule, add explicit hydrogen, assign partial charges using AM1-BCC and save the resulting molecule to a `MOL2` file. The ...
```
Comment on where the structure came from (provided by Lyle Isaacs, from a crystal structure -- but we are not releasing that crystal structure).

bergazin replied:
Not released until the challenge ends, correct (ie that CIF file is the crystal structure, right?)

davidlmobley replied:
Yes, right. I would just explain all of that here.

davidlmobley replied:
(See remarks I added).

davidlmobley commented on host_guest/Isaacs_clip/host_guest/README.md:
```diff
@@ -0,0 +1,9 @@
+- Provided is the Trimertrip host for this challenge in PDB, Tripos MOL2, and SDF file format (trimertrip.mol2, trimertrip.sdf, trimertrip.pdb).
+- Trimertrip host files were created as follows:
+  - Original starting structure (in `CIF` file format, which will be added here once the challenge ends) was converted to a PDB using OpenEye toolkits. Water, a guest, a single Na+ ion, and connectivity records were manually removed. OpenEye toolkit was used to normalize the molecule, add explicit hydrogen, assign partial charges using AM1-BCC and save the resulting molecule to a `MOL2` file. The ...
+  - File creation: The `pytraj` toolkit was used to remove the solvent (water, Na+ and Cl-) and create a `MOL2` file, then the `antechamber` toolkit was used to convert GAFF2 atom types to SYBYL/Tripos atom type, the resulting `MOL2` file was read into OpenEye and written out to a new standards-compliant `MOL2` file, this `MOL2` file was then used to generate a SDF and PDB file.
```
What package is `pytraj` part of? Is this part of AMBERTOOLS?

davidlmobley replied:
And, last bit, "this `mol2` ... used to generate..." HOW was it used? OpenEye tools? Or a different mechanism.

bergazin replied:
pytraj is the python interface of cpptraj (which is apart of AmberTools)

davidlmobley commented on host_guest/Isaacs_clip/host_guest/README.md:
```diff
@@ -0,0 +1,9 @@
+  - Original starting structure (in `CIF` file format, which will be added here once the challenge ends) was converted to a PDB using OpenEye toolkits. Water, a guest, a single Na+ ion, and connectivity records were manually removed. OpenEye toolkit was used to normalize the molecule, add explicit hydrogen, assign partial charges using AM1-BCC and save the resulting molecule to a `MOL2` file. The ...
+  - File creation: The `pytraj` toolkit was used to remove the solvent (water, Na+ and Cl-) and create a `MOL2` file, then the `antechamber` toolkit was used to convert GAFF2 atom types to SYBYL/Tripos atom type, the resulting `MOL2` file was read into OpenEye and written out to a new standards-compliant `MOL2` file, this `MOL2` file was then used to generate a SDF and PDB file.
+
+- Guests are codenamed from `g1` to `g15` and are provided in PDB, Tripos MOL2, and SDF file format. Guest file preparation is provided in the `input_maker.ipynb` jupyter notebook in this folder. For guest `g11` a random stereoisomer was chosen.
```
You want to explain WHY a random stereoisomer was acceptable -- that experiment was done on a racemic mix but because the host is achiral, the chirality of the guest is unimportant for binding.

bergazin replied:
Added.

bergazin commented:
Listing some changes made so far
- For SMILES strings and names in guests.csv and guests.ism, I switched them to a colon-delimited file where the ID comes last. Also added a textfile (.txt) that has this format so that it’s consistent with the smiles textfiles in the Isaacs/Gibb folders.
-Removed the ions from the Isaacs SMILES strings. Regenerated the input pdb/mol2/sdf files to make sure the ions weren’t included. Checked the Gibb SMILES textfile for ions, didn’t see any included so no changes made. 
- Removed “tentative” from Isaacs/Gibbs readme and updated file names (ie `SAMPL7_tentative.cdx` --> `SAMPL7.cdx`) in directories/subdirectories

davidlmobley commented on host_guest/GDCC_and_guests/README.md:
```diff
@@ -28,11 +28,11 @@ Given the identity of these guests, it is likely we will be able to have *some*
 ### Additional technical details
 
-Tentatively, buffer conditions will be 10 mM sodium phosphate, but this will be confirmed as data collection begins. The plan is to gather data in triplicate with fresh solutions of host and guest on each occasion.  Also, all hosts samples are probed by one of either two ways to determine the waters of hydration in each sample.  This can be as high so this analysis is a requirement to avoid bad d ...
+Buffer conditions are expected to be 10 mM sodium phosphate, but this will be confirmed as data collection begins. The plan is to gather data in triplicate with fresh solutions of host and guest on each occasion.  Also, all hosts samples are probed by one of either two ways to determine the waters of hydration in each sample.  This can be as high so this analysis is a requirement to avoid bad dat ...
```
This is good for now, but I'll e-mail hjim.

davidlmobley commented on host_guest/Isaacs_clip/host_guest/README.md:
```diff
@@ -1,9 +1,9 @@
 # Host
 
-- Provided is the Trimertrip host for this challenge in PDB, Tripos MOL2, and SDF file format (trimertrip.mol2, trimertrip.sdf, trimertrip.pdb).
+- Provided is the Trimertrip host for this challenge in PDB, Tripos MOL2, and SDF file format (trimertrip.mol2, trimertrip.sdf, trimertrip.pdb). The structure was provided by Lyle Isaacs, from a crystal structure. The crystal structure will not be released. The original starting structure will be added here once the challenge ends
```
Needs clarification as to "the structure". Specifically, I'd say something like "The structure was originally provided by Lyle Isaacs as a CIF file from crystallography. However, to avoid biasing results we are not releasing this structure at present, and instead equilibrated a structure of the host (`[detailed elsewhere in this repo](link)`) which we are providing here in sdf, mol2 and PDB formats. The original structure from crystallography will eventually be provided here once the challenge concludes."

bergazin replied:
Updated!

davidlmobley commented on host_guest/Isaacs_clip/host_guest/README.md:
```diff
@@ -1,9 +1,9 @@
 - Trimertrip host files were created as follows:
   - Original starting structure (in `CIF` file format, which will be added here once the challenge ends) was converted to a PDB using OpenEye toolkits. Water, a guest, a single Na+ ion, and connectivity records were manually removed. OpenEye toolkit was used to normalize the molecule, add explicit hydrogen, assign partial charges using AM1-BCC and save the resulting molecule to a `MOL2` file. The ...
-  - File creation: The `pytraj` toolkit was used to remove the solvent (water, Na+ and Cl-) and create a `MOL2` file, then the `antechamber` toolkit was used to convert GAFF2 atom types to SYBYL/Tripos atom type, the resulting `MOL2` file was read into OpenEye and written out to a new standards-compliant `MOL2` file, this `MOL2` file was then used to generate a SDF and PDB file.
+  - File creation: The `pytraj` toolkit (the python interface of `cpptraj`, which is apart of `AmberTools`) was used to remove the solvent (water, Na+ and Cl-) and create a `MOL2` file, then the `antechamber` toolkit was used to convert GAFF2 atom types to SYBYL/Tripos atom type, the resulting `MOL2` file was read into OpenEye and written out to a new standards-compliant `MOL2` file, this `MOL2`  ...
```
"a part", not "apart". Those two mean almost opposite things, oddly.

bergazin replied:
Changed.
